services/core-api/tests/status_code_gen.py

```diff
@@ -35,6 +36,8 @@ def RandomMineRegionCode():
 def RandomPermitStatusCode():
     return random.choice([x.permit_status_code for x in db.session.query(PermitStatusCode).all()])
 
+def RandomExemptionFeeStatusCode():
+    return random.choice([x.exemption_fee_status_code for x in db.session.query(ExemptionFeeStatus).all()])
```

Comment:
Do we want to generate records with inactive exemption fee statuses? If not, maybe use ExemptionFeeStatus.get_active().

Reply:
Good observation, changed